vnext/ReactUWPTestApp/Themes/Generic.xaml

```diff
@@ -0,0 +1,49 @@
+<ResourceDictionary
+    xmlns="http://schemas.microsoft.com/winfx/2006/xaml/presentation"
+    xmlns:x="http://schemas.microsoft.com/winfx/2006/xaml"
+    xmlns:local="using:ReactUWPTestApp">
+
+  <Style x:Key="AutomationHelperStyle" TargetType="Control">
+  </Style>
+
+  <Style TargetType="local:TestFrame" >
+    <Setter Property="Template">
+      <Setter.Value>
+        <ControlTemplate TargetType="local:TestFrame">
+          <Grid x:Name="RootGrid">
+            <Grid.RowDefinitions>
+              <RowDefinition Height="Auto"/>
+              <RowDefinition />
+            </Grid.RowDefinitions>
+
+            <Grid x:Name="TestFrameBar" VerticalAlignment="Top" HorizontalAlignment="Left">
+              <StackPanel Orientation="Horizontal">
+                <Button x:Name="HomeButton" AutomationProperties.AutomationId="__HomeButton">
+                  <SymbolIcon Symbol="Home"/>
+                </Button>
+                <Button x:Name="BackButton" 
```

Comment:
Did you consider making these buttons in RN rather than native only (like RNTester)?  This would more easily allow us to bring this stuff to other platforms.

Reply:
Yes, it would be great to have this mimic the RNTester detox tests as closely as possible.